composer.json

```diff
@@ -1,10 +1,14 @@
 {
-    "name": "wysow/postfinance",
+    "name": "balidra/postfinance",
```

Comment:
This is not possible, you cannot change the package name as Packagist already knows it as `wysow/postfinance` and it should not be duplicated...